src/style.scss

```diff
@@ -3,30 +3,178 @@
+  flex-direction: column;
+  align-items: center;
+
+  @supports (display: grid) {
+    display: grid;
+    grid-template-columns: auto;
+    grid-template-rows: 1.625rem 1.25rem;
+  }
+  
+  padding-top: 1.375rem;
+  height: 5.625rem;
+  min-width: 4.25rem;
+
+  font-family: var(--auro-font-family-default);
+  font-weight: var(--auro-text-body-default-weight);
+  background: var(--auro-color-base-gray-100);
+  border: 2px solid;
+  border-color: var(--auro-color-base-gray-200);
+  border-radius: 2.5rem;
+  cursor: pointer;
+
+  @include auro_breakpoint--sm {
+    font-size: var(--auro-text-body-size-default);
+    grid-template-rows: 1.75rem 1.5rem;
+    padding-top: 1.625rem;
+    height: 6.5rem;
+    min-width: 5rem;
+  }
+
+  @media (hover: hover) {
+    &:hover {
+      background: var(--auro-color-border-active-on-light);
+      border-color: var(--auro-color-border-active-on-light);
+
+      .child {
+        color: var(--auro-color-base-white);
+      }
+    }
+  }
+
+  &:active {
+    background: var(--auro-color-ui-active-on-light);
+    border-color: var(--auro-color-ui-active-on-light);
+    transform: scale(0.95);
+  }
+
+  &:disabled {
+    background: none;
+    border-color: transparent;
+    cursor: not-allowed;
+    transform: unset;
+
+    .child {
+      pointer-events: none;
+      color: var(--auro-color-text-secondary-on-light);
+    }
+  }
+}
+
+.pane--selected {
+  background: var(--auro-color-ui-active-on-light);
+  border-color: var(--auro-color-ui-active-on-light);
 }
+
+.pane--priced {
+  height: 6rem;
+  min-width: 4.5rem;
+  border-radius: 2.75rem;
+  grid-template-rows: 1.375rem 1.25rem 1.25rem;
+  padding-top: var(--auro-size-md);
+
+  @include auro_breakpoint--sm {
+    height: 7.5rem;
+    min-width: 5.5rem;
+    padding-top: var(--auro-size-lg);
+    grid-template-rows: 1.5rem 1.25rem 1.625rem;
+  }
+
+  .day-of-week {
+    color: var(--auro-color-text-primary-on-light);
+    font-size: var(--auro-text-body-size-default);
+    line-height: var(--auro-text-body-height-default);
+
+    @include auro_breakpoint--sm {
+      font-size: var(--auro-text-body-size-lg);
+      line-height: var(--auro-text-body-height-lg);
+    }
+  }
+}
+
+.pane:active:not([disabled]),
+.pane--selected {
+  .child {
+    color: var(--auro-color-base-white);
+  }
+}
+
+.day-of-week {
+  color: var(--auro-color-ui-default-on-light);
+  font-size: var(--auro-text-body-size-lg);
+  line-height: var(--auro-text-body-height-lg);
+
+  @include auro_breakpoint--sm {
+    font-size: 1.25rem;
+    line-height: 1.75rem;
+    letter-spacing: var(--auro-text-heading-default-spacing);
+  }
+}
+
+.date {
+  color: var(--auro-color-text-secondary-on-light);
+  font-size: var(--auro-text-body-size-sm);
+  line-height: var(--auro-text-body-height-sm);
+
+  @include auro_breakpoint--sm {
+    font-size: var(--auro-text-body-size-default);
+    line-height: var(--auro-text-body-height-default);
+  }
+}
+
+// Separate the words when read by screen reader
```

Comment:
See the videos in Teams.

Reply:
I originally added this because NVDA was failing to separate the words in the button. For whatever reason this is no longer happening, so I went ahead and removed this.